Apps/WebClient/src/ClientApp/public/index.html

```diff
@@ -59,8 +60,24 @@
             window.snowplow("trackPageView");
         </script>
         <!-- Snowplow stop plowing -->
+        
+        <script type="text/javascript">
+            function detectBrowser() {
+                const body = document.getElementsByTagName("BODY")[0];
+                const unsupportedBrowser = document.getElementById("unsupported-browser");
+                if (Modernizr.es6number) {          
+                    body.removeChild(unsupportedBrowser);
+                }
+                else {          
+                    body.removeChild(document.getElementById("app-root"));
+                    const unsupportedBrowserMsg = "Health Gateway is not compatible with this browser. Please use another browser.";
```

Comment:
if the contents of the div are going to be removed, why not set them there instead of programmatically changing it?

Reply:
sounds like a better approach, I can chance it

Reply:
doesn't seem to work as VueJS is automatically looking for its app-root div. I leave it as is for now if you don't mind
![image](https://user-images.githubusercontent.com/48332333/93502548-bb247680-f8cb-11ea-8fad-8a80e640630f.png)

Reply:
I think it will give that errors based on ```body.removeChild(document.getElementById("app-root"));```  What I mean is that "Health Gateway is not compatible with this browser. Please use another browser." should be  in the ```  <div id="unsupported-browser"></div>``` already

Reply:
oh I see, good idea. I 'm fixing it now

Reply:
btw, in IE, I'm seeing 2 div (loading... and unsupported one) at the same time before the loading... is removed. I will leave them as they are then?

Reply:
both should be there, since we dont know when does the vue loading happen vs the script execution. you can also add a display none to both and remove it on the script